### DISCUSSION TITLE
ZepetoScript 캐시를 비우는 방법이 궁금합니다

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#638] [5. Q&A]
Typescript로 작성한 ZepetoScript 를 사용중 참조 에러가 나는 경우가 매우 빈번한데
표면상으로는 변수의 참조를 찾지 못해 undefined로 출력이 됩니다.

반복적으로 코드를 다시 봐도 원인을 찾지 못하였고
프로젝트를 다른 컴퓨터에서 Pull받아서 테스트하면 정상적으로 동작합니다.
혹은 새 프로젝트를 생성해서 에셋들을 복사해도 정상적으로 동작합니다.

다만 문제가 발생할 때마다 이렇게 조치하기에는 개발이 너무 지연되는 상황입니다.

아마 컴파일된 Typescript 캐시들이 어딘가에 존재해서 이런 현상이 발생하는 것으로 보입니다만...
캐시가 어떤 경로에 존재하는지, 어떻게 비울 수 있는지 알려주시면 감사하겠습니다

-------------------------------------------

추가로 문제가 발생했던 코드의 이미지를 첨부하였습니다. 참고해주시면 감사하겠습니다
![image](https://user-images.githubusercontent.com/100827088/163952511-0d634975-b6f2-414b-ae7b-360bb6058ced.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 참조 에러가 발생할 경우 아래 2가지 방법으로 진행 부탁 드립니다. 

1. ZEPETO - Script - All Rebuild
2. 불러오지 못하는 TS 파일을 선택하여 inspector 에서 force rebuild
<img width="532" alt="image (1)" src="https://user-images.githubusercontent.com/100118650/164190455-82ef01ff-1b85-4952-80a1-d3c2aab8ad35.png">

두 가지 방법으로도 해결되지 않을 경우, 개발 환경(OS, world version)에 대한 정보와 함께 댓글 달아주세요. 감사합니다.
